Build errors about documentation headers when building Tests

Here is the error:

```
In file included from /Users/gbolsinga/code/Texture/Tests/ASCenterLayoutSpecSnapshotTests.mm:10:
In file included from /Users/gbolsinga/code/Texture/Tests/ASLayoutSpecSnapshotTestsHelper.h:10:
In file included from /Users/gbolsinga/code/Texture/Tests/ASSnapshotTestCase.h:10:
In file included from /Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestCase.h:12:
/Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestController.h:92:9: error: parameter 'error' not found in the function declaration [-Werror,-Wdocumentation]
 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~
/Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestController.h:92:9: note: did you mean 'errorPtr'?
 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~
        errorPtr
/Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestController.h:105:9: error: parameter 'error' not found in the function declaration [-Werror,-Wdocumentation]
 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~
/Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestController.h:105:9: note: did you mean 'errorPtr'?
 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~
        errorPtr
/Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestController.h:115:9: error: parameter 'view' not found in the function declaration [-Werror,-Wdocumentation]
 @param view The view or layer to snapshot.
        ^~~~
/Users/gbolsinga/code/Texture/Pods/Headers/Public/FBSnapshotTestCase/FBSnapshotTestController.h:119:9: error: parameter 'error' not found in the function declaration [-Werror,-Wdocumentation]
 @param error An error to log in an XCTAssert() macro if the method fails (missing reference image, images differ, etc).
        ^~~~~
4 errors generated.

```

This is fixed in the latest FBSnapshotTestCase, but it has changed ownership, bumped versions and moved past the versions supported by Texture. I think we can disable this warning locally in `ASSnapshotTestCase`.

--- a/Tests/ASSnapshotTestCase.h
+++ b/Tests/ASSnapshotTestCase.h
@@ -7,7 +7,11 @@
 //  Licensed under Apache 2.0: http://www.apache.org/licenses/LICENSE-2.0
 //
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
 #import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+#pragma clang diagnostic pop
+
 #import "ASDisplayNodeTestsHelper.h"
 
 @class ASDisplayNode;

--- a/Tests/ASTableViewTests.mm
+++ b/Tests/ASTableViewTests.mm
@@ -8,7 +8,10 @@
 //
 
 #import <XCTest/XCTest.h>
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
 #import <JGMethodSwizzler/JGMethodSwizzler.h>
+#pragma clang diagnostic pop
 
 #import <AsyncDisplayKit/AsyncDisplayKit.h>
 #import <AsyncDisplayKit/ASTableView.h>

--- a/Tests/ASTextKitTests.mm
+++ b/Tests/ASTextKitTests.mm
@@ -10,7 +10,10 @@
 #import <UIKit/UIKit.h>
 #import <XCTest/XCTest.h>
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
 #import <FBSnapshotTestCase/FBSnapshotTestController.h>
+#pragma clang diagnostic pop
 
 #import <AsyncDisplayKit/ASTextKitAttributes.h>
 

